pyflap.py

```diff
@@ -26,23 +25,74 @@
     for file in ASSETS.glob("bird/*.png")
 ]
 
+class Bird():
```

Comment:
Oops, looks like some remains from where I subclassed `pygame.Rect` lived on on this line